e2e/nextjs-app/test/ingest.spec.ts

```diff
@@ -25,21 +25,28 @@ describe('Ingestion & query on different runtime', () => {
     const startTime = new Date(Date.now()).toISOString();
     // call route that ingests logs
     const resp = await fetch(process.env.TESTING_TARGET_URL + '/api/lambda');
-    expect(resp.status).toEqual(200);
+    expect(200).toEqual(resp.status);
```

Comment:
nit: this should be changed back since all the other asserts are the other way around.